frontend/src/lib/i18n/locales/en.json

```diff
@@ -309,7 +309,7 @@ the [Linguistics Institute at Payap University](https://li.payap.ac.th/) in Chia
   },
   "form": {
     "password": {
-      "allowed_characters": "Only letters, numbers and dashes (-) are allowed",
+      "forbidden_characters": "The symbols & (ampersand), + (plus), and % (percent) can cause problems in passwords",
```

Comment:
That's what I would shorten it to.
I don't think the names of the symbols are helpful for a user
```suggestion
      "forbidden_characters": "The symbols &, +, and % are not allowed",
```

Reply:
Not all fonts represent symbols in a way that's easy to read, so I think spelling them out is helpful.